lib/tables.c

```diff
@@ -482,6 +545,75 @@ node_table_dump_text(node_table_t *self, FILE *out)
     return ret;
 }
 
+int
+node_table_load_text(node_table_t *node_table, FILE *file)
+{
+    int ret = MSP_ERR_FILE_FORMAT;
+    int err;
+    size_t k;
+    size_t MAX_LINE = 1024;
```

Comment:
Since we're accepting arbitrary metadata here, I don't think a fixed alloc is going to work. We're going to need someway to read in an arbitrary line length.

Reply:
Scratch that, readline reallocs the buffer as necessary.